draft-dejong-remotestorage-06.txt

```diff
@@ -109,15 +108,14 @@ Internet-Draft              remoteStorage                  November 2015
           and contents
 
        *  PUT a document: store a new version, its content type, and
-          contents, conditional on the current version
+          contents
 
-       *  DELETE a document: remove it from the storage, conditional on
-          the current version
+       *  DELETE a document: remove it from the storage
 
        *  HEAD a folder or document: like GET, but omitting the response
           body
 
-    The exact details of these four actions are described in this
+    The semantics of these four HTTP verbs are described in this
```

Comment:
The HTTP verb is described in the HTTP spec. This is about the RS-specific action of storing, getting and deleting. So I like the old one better.